### PR TITLE
Add Google Groups Documentation

### PR DESCRIPTION
**Pull request by CannonLock**
https://osg-htc.org/web-preview/preview-google-groups-doc

### CODE REVIEW

File: _community/google-groups.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: Google Groups - Managing List Membership
```
Comment (osg-cat) [inline _community/google-groups.md:2]: Overall, I think we should really de-emphasize the Google Groups thing on this page. We are really just using them as glorified mailing lists, and so let’s stick to just that one concept.

So, for a title, how about “Manage Your OSG List Memberships” or something?

File: _community/google-groups.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+layout: text-optimized-width
+
+---
+
+# {{ page.title }}
+
+This documents how to join and leave a Google Group via webpage and email.
```
Comment (osg-cat) [inline _community/google-groups.md:9]: How about? “The OSG Consortium is moving to using Google Groups for managing mailing lists that end in `osg-htc.org`. This page describes how to manage your membership in these lists.”

File: _community/google-groups.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+If you have any issues following the guides below please reach out to us at [support@osg-htc.org](mailto:support@osg-htc.org) with the email you would like subscribed and the list you would like to subscribe too.
```
Comment (osg-cat) [inline _community/google-groups.md:11]: This comment, as written, is specific to subscribing. Let’s put a more general statement here and then maybe a detailed reminder under the Join section.

File: _community/google-groups.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+##### Contents
```
Comment (osg-cat) [inline _community/google-groups.md:13]: Is there no way to have this as a sidebar? There is a lot of wasted space on this page, IMO.

File: _community/google-groups.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+- [Joining a list via webpage](#joining-a-list-via-webpage)
+- [Leaving a list via webpage](#leaving-a-list-via-webpage)
+- [Joining a list via email](#joining-a-list-via-email)
+- [Leaving a list via email](#leaving-a-list-via-email)
+
+## Joining a list via webpage
```
Comment (osg-cat) [inline _community/google-groups.md:20]: Ok, so just a rant here: This is not a good way to write procedural documentation. The user just wants to join or leave. How should they know whether it is better to use a webpage or email? Now they are confused, and rage quit.

Focus always, always, always on the user’s goal, and help them accomplish their goal as painlessly as possible. So there should be one heading for joining. One heading for leaving. Can users do anything else meaningful? See a history of messages for groups that allow it? Maybe we should generalize the page a bit, although I will not complain if we focus for now on joining and leaving.

File: _community/google-groups.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+To join a group via the Google Groups UI you should:
```
Comment (osg-cat) [inline _community/google-groups.md:22]: First step is to pick a method. Why prefer one over the other? Is one better? Easier? Faster? If there is no obvious winner, then _you_ pick one as the default and just say that the other method may work if the first does not.

File: _community/google-groups.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Login to google account you wish to subscribe.
```
Comment (osg-cat) [inline _community/google-groups.md:24]: 1. Choose an email address to subscribe
2. If that email address is associated with a Google Account, then… (and skip to Step N when done)
3. If not, then…

File: _community/google-groups.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+    - If you don't wish to subscribe via Google account see [below](#joining-a-list-via-email).
+1. Navigate to the groups page at ```https://groups.google.com/u/1/a/osg-htc.org/g/[Group Name]```.
+    - For example group **osg-group**: <br> ```https://groups.google.com/u/1/a/osg-htc.org/g/osg-group```.
+1. Click on the button marked "Join Group" (Shown Below).
+
+#### Notes:
+- This method requires the group is open to the public and a Google Associated email address
```
Comment (osg-cat) [inline _community/google-groups.md:31]: Don’t put information like this at the end! Put items in the exact order in which the user needs to deal with them.

File: _community/google-groups.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+![Arrow pointing to Join Groups Button in Google Groups UI](../assets/images/Google_Group_Join_UI.png){: .img-fluid }
+
+## Leaving a list via webpage
```
Comment (osg-cat) [inline _community/google-groups.md:35]: All previous comments for sequencing and flow apply here, too.

File: _community/google-groups.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Login to subscribed email account via Google.
+2. Select group from the ["My Groups" page](https://groups.google.com/my-groups).
+3. Click "My membership settings".
+4. Click "Leave Group" button found in the same location as the "Join Group" button above.
+
+## Joining a list via email
+
+To join a Google Group via email, email ```[Group Name]+subscribe@osg-htc.org```.
```
Comment (osg-cat) [inline _community/google-groups.md:44]: Again, de-emphasize the Google Groups aspect. “To join an OSG Consortium mailing list:”
1. Send email to ...
    Note: Your email does not need a subject or body

File: _community/google-groups.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+For example, to join the group called osg-group, email ```osg-group+subscribe@osg-htc.org```.
```
Comment (osg-cat) [inline _community/google-groups.md:46]: Procedural steps should be numbered, always.

File: _community/google-groups.md [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+
+In order for us to complete the request, please reply to this email or click
+[the button] below.
+
+[Join This Group]
+```
+
+#### Notes:
+
+- Always reply to the email to join. Clicking the button is inconsistent and will sometimes give an error of "you do not have permissions to access this group".
```
Comment (osg-cat) [inline _community/google-groups.md:62]: This should be a procedural step, not an extraneous note.

File: _community/google-groups.md [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+
+- You should send the subscribing message from the email address you want to receive messages. The above instructions need only be used for those without direct access to Google Groups at https://groups.google.com/a/wisc.edu/forum/#!myforums.
```
Comment (osg-cat) [inline _community/google-groups.md:64]: This information _must_ be associated with the step itself!

File: _community/google-groups.md [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+
+- For some groups, you may need to confirm the subscription request, as delineated above, while other with more restricted membership will require the group owner to approve your subscription request before you can join.
```
Comment (osg-cat) [inline _community/google-groups.md:66]: This is fine as a note.

File: contact.md [hunk 15]
```diff
@@ -13,7 +13,9 @@ Email: <mailto:osg-contact@opensciencegrid.org>
 * Email: <mailto:support@osg-htc.org>
 * OSG Connect Support: <mailto:support@osgconnect.net> (for users of [OSG Connect](https://support.opensciencegrid.org))
 
-
 ### For OSG policies, executive information
 Email: [Frank Wuerthwein](mailto:fkw@ucsd.edu) (OSG Executive Director)
 
+### For help joining an OSG Mailing list
```
Comment (osg-cat) [inline contact.md:19]: Let’s be sure to use consistent terminology and concepts here

**Review by osg-cat — CHANGES_REQUESTED** [review]
Cannon, this is a good start but needs some work.

File: _community/google-groups.md (hunk 7 above)
Comment (CannonLock) [reply _community/google-groups.md]: I went a slightly different direction with this and decided to fork them after the first step into two different methods. Neither of the method share ending steps so it made sense to just completely separate them.

**Comment by CannonLock** [comment]
Hey Tim, thanks for the notes. A lot of great points I will make sure to remember in the future!

**Review by osg-cat — APPROVED** [review]
Cannon, I think that this is getting close. How about merging it in, and then I’ll make one or more PRs and you can cherry-pick from them?

— Tim

**Pull request merged**